src/Swashbuckle.AspNetCore.SwaggerGen/SwaggerGenerator/ISchemaGenerator.cs

```diff
@@ -12,6 +12,7 @@ public interface ISchemaGenerator
             SchemaRepository schemaRepository,
             MemberInfo memberInfo = null,
             ParameterInfo parameterInfo = null,
-            ApiParameterRouteInfo routeInfo = null);
+            ApiParameterRouteInfo routeInfo = null,
+            bool isEffectiveTypeNeeded = true);
```

Comment:
This is a binary-breaking change we cannot make outside of a major version (and then only if we deem it worthy of the break).